.vale/styles/RedHat/Headings.yml

```diff
@@ -46,78 +50,79 @@ exceptions:
   - Containerfile
   - Containerfiles
   - Cookiecutter
-  - CR
-  - CRD
-  - CRDs
-  - CRs
   - Ctrl
   - Cygmon
+  - DNS
   - DaemonSet
   - Datadog
   - Dev
   - DevOps
   - DevWorkspace
   - Dex
-  - DNS
   - Docker
   - Dockerfile
   - Dockerfiles
   - Dotnet
-  - Eclipse
   - Dyninst
+  - Eclipse
+  - Electron
   - Endevor
   - Endian
-  - eServer
   - Esprima
   - Exif
   - Fabrice
   - Failsafe
   - Fernflower
   - Flathub
   - Flatpak
+  - Flatpak Builder
```

Comment:
Should `Flatpak Builder` be declared in the CaseSensitive.yml and fixtures too to catch lowercase instances of `flatpak builder`? 

(I'm assuming `Flatpak builder` is valid when talking in a general sense about the builder for Flatpak.)

Reply:
I have no idea